test/fixtures/basic.styl

```diff
@@ -1,2 +1,2 @@
 p
-  display: flex
+  display flex
```

Comment:
Why remove the colon here? I prefer using colons even within stylus to make the key/value split clear and the code more understandable.

Reply:
I agree on readability, the change was mostly because stylus allows it and to make sure it's correctly transpiled.

Reply:
This library doesn't need to test stylus' features - the fact that there are no brackets is already enough to separate it from normal css and ensure it is being transpiled 😀